spec/fixtures/gradle_with_kotlin/gradle-dependencies-q.txt

```diff
@@ -0,0 +1,312 @@
+
```

Comment:
And this file was the output of `gradle dependencies -q` on the above build.gradle.kts.